### ISSUE TITLE
Ethan Bennett Final Project

### ISSUE THREAD

**Issue by ekb53** [b-ivey/final-project#11]
https://github.com/ekb53/ekb53.github.io/tree/master/Web%20Design%20Final

**Comment by ekb53** [comment]
I followed all the steps to connect my domain (ethanbennett.me) through NameCheap, and the CNAME file is in my repository, but I'm still getting a 404 error. Please let me know if there's something else I should do. Thank you

**Comment by b-ivey** [comment]
The most obvious issue for the 404 error code is the CNAME file, which is supposed to be in the same repository as your site files. Your stylesheet should be in its own folder and titled stylesheet.

Finally, this was submitted past the 11:59 pm deadline, so late points will be added. I'll email grades and a critique today.

**Comment by ekb53** [comment]
I completely forgot about the time zone difference, my apologies. The CNAME file is in the same repository as the web files. Do you mean it needs to be in the same folder?

Thank you
________________________________

**Comment by b-ivey** [comment]
Yes, they all need to be in the same folder. In order for the domain name server to talk to GitHub, you’ll have to take the site files OUT of the Web Design Final folder.
